src/polygon.c

```diff
@@ -556,6 +558,57 @@ spoly_segment(SLine *sl, const SPOLY *poly, int32 i)
 	}
 }
 
+static bool
+poly_get_point(SPoint *sp, const SPOLY *poly, int32 i)
```

Comment:
I propose to rename the function name to spoly_get_point to be consistent with the rest of the code. spoly prefix is used in names of other functions.